### PR TITLE
Added support for Tesla drivers

### PR DESCRIPTION
**Pull request by LawfulHacker**
This commit adds support for Tesla drivers which are used even on desktops when you install the Cuda SDK.
It was tested on a GeForce GTX 1050Ti and a GeForce GTX 1650, using Ubuntu 19.04, after installation of the Cuda SDK.

### CODE REVIEW

**Comment by LawfulHacker** [comment]
Apparently this failed check is wrong as it is looking for a file which is generated by the _Makefile_ based on _org.freedesktop.Platform.GL.nvidia.json.in_, please check before approving this PR.

**Comment by TingPing** [comment]
Yeah ignore the bot on this repo.

File: versions.sh [hunk 1]
```diff
@@ -6,9 +6,10 @@ VULKAN_VERSIONS="418.52.14 418.52.10 418.52.07 418.52.05 418.52.03 418.49.04 418
 SHORT_LIVED_VERSIONS="415.27 415.25 415.23 415.22 415.18 396.54 396.51 396.45 396.24"
 LONG_LIVED_VERSIONS="430.34 430.26 430.14 418.74 418.56 418.43 410.104 410.93 410.78 410.73 410.66 410.57"
 LEGACY_VERSIONS="340.107 340.106 340.104 340.102 340.101 304.137 304.135 304.134"
+TESLA_VERSIONS="418.67 418.40.04 410.79 410.72"
```
Comment (TingPing) [inline versions.sh]: Is there a download page that keeps a list of versions for this like the other drives have?

Comment (LawfulHacker) [reply versions.sh]: This is the best match I found:
https://docs.nvidia.com/datacenter/tesla/index.html

Comment (TingPing) [reply versions.sh]: Guess that works, mind throwing a comment above this line for future version bumpers.

Comment (LawfulHacker) [reply versions.sh]: Ok, will do that and also add a note to not include versions that also exists on others repositories, as they are downloaded from that location and not from the tesla path (for example 410.104).

**Pull request merged**